docs/use-cases/portfolio-management/README.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases: Portfolio Management and Custom Reporting
+  priority: 999
+description: Step-by-step instruction about portfolio management and custom reporting
+robots: index, follow
+keywords: help, documentation, use-cases, portfolio management
+---
+# Use Cases:
+
+# Portfolio Management and Custom Reporting Options
```

Comment:
Please don't use multiple level-1 headlines after another, better combine them 
`# Use Cases: Portfolio Management and Custom Reporting Options`